about.html

```diff
@@ -0,0 +1,137 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About CC Bootcamp</title>
+    <link href="css/main.css" rel="stylesheet">
+    <script src="scripts/main.js" defer></script>
+</head>
+<body class="mainContainer">
+    <header class="header">
+        <nav class="navbar navbar-black social desktop desktop-flex">
+            <ul class="nav nav-start">
+                <li>
+                    <a rel="noopener noreferrer" href="https://github.com/JohnFTitor" target="_blank" class="social-icon"><img src="assets/icons/github.svg" alt="Github Link"></a>       
+                </li>   
+                <li>
+                    <a rel="noopener noreferrer" href="https://twitter.com/johnftitor" target="_blank" class="social-icon"><img src="assets/icons/twitter.svg" alt="Twitter Link"></a>       
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">Spanish</a>   
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">My Page</a>   
+                </li>
+                <li>
+                    <a rel="noopener noreferrer" href="#">Log Out</a>   
+                </li>
+            </ul>
+        </nav>
+        <nav class="navbar">
+            <ul class="nav">
+                <li class="mobile"><button type="button" id="menu"><img src="assets/icons/menu.svg" alt="Menu Button"></button></li>
+                <li class="desktop"><a rel="noopener noreferrer" href="index.html"><img src="assets/images/logo.svg" alt="Home Page"></a></li>
+                <li class="desktop">
+                    <ul class="nav nav-links">
+                        <li >
+                            <a rel="noopener noreferrer" href="about.html"> About </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="index.html#main-program"> Program </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> Join </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> Sponsor </a>
+                        </li>
+                        <li>
+                            <a rel="noopener noreferrer" href="#"> News </a>
+                        </li>
+                        <li>
+                            <button type="button" class="interaction interaction-alternative">CC Campaing</button>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="content">
+        <section id="about">
+            <div class="container about about-principal">
+                <p class="title title-description mobile">"Welcome! Future Developer"</p>
+                <h1 class="title title-header">CREATIVE COMMONS BOOTCAMP SUMMER 2022</h1>
```

Comment:
- [ ] Please, I would suggest you use only one `h1` tag per page for optimal SEO. A standard webpage maintains a single `h1` tag in the entire file. You can read more [here](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).